project-icp/bee-pollinator-app
# Beekeepers fetch endpoint accepts data on POST request

## Overview

Replace the dummy data the endpoint used with real data passed in on the POST request.
The data format should be compatible with the front end whereby we will pass in the leaflet geom unmodified `{lat: 2, lng: 1}`, the "RASTERS" constant for the filenames, and eventually also the forage range as 3k or 5k (subj to change). We don't have all the Rasters yet from the client, so file failures will return `0` for the data rather than fail for files that can't be found.

Connects #308 

### Demo

After submitting the JSON provided in the testing section
![screen shot 2018-11-06 at 11 00 44 am](https://user-images.githubusercontent.com/10568752/48076508-4f433400-e1b3-11e8-8fad-a8e3ddedfdf9.png)


## Testing Instructions

Submit data with the required params to the fetch endpoint at http://localhost:8000/beekeepers/fetch/
The coordinates should be in Pennsylvania

```
{
    "forage_range": "3k",
    "location": {
        "lat": 40.741,
        "lng": -76.123
    },
    "filenames": {
        "PESTICIDE": "pesticide",
        "FORAGE_SPRING": "forage_spring",
        "FORAGE_SUMMER": "forage_summer",
        "FORAGE_FALL": "forage_fall"
    }
}
```
